### ISSUE TITLE
Instruction on how to run azure-xplat-cli with docker in the release notes of azure-xplat-cli and maybe integrate a Dockerfile there

### ISSUE THREAD

**Issue by colrack** [Azure/azure-cli-docker#23]
Following discussion on PR #22

@ahmetalpbalkan you are right this repo maybe make sense.

However what I can report is this repo not been updated for a while (0.9.11, 0.9.12 and 0.9.13 are missing as for now); I feel like if azure-xplat-cli team would be responsible of updating this would be nice.

I think also having a Dockerfile in azure-xplat-cli to build the current version would be great, so we can clone the repo, build the image and run without having node on the host system.

For docker releases I mean to have something like `docker run -it microsoft/azure-cli:X.X.X` in the release notes.

What do you think about?
Thanks

**Comment by ahmetalpbalkan** [comment]
Yes, normally the team is responsible for releasing the versions of this image, I am voluntarily submitting the pull requests. Folks may be on vacation during this timeframe, that's what I attribute this silence to. :-)

For your other proposal, if you would like to to run the master branch of azure-xplat-cli without installing node.js, you will need to have a Dockerfile in azure-xplat-cli repo root as you said. However it's a bit outside of scope for this repo, please feel free to propose a Dockerfile over at that repo. :-) Or alternatively just copy this Dockerfile over there and change as you see fit for your personal use.

**Comment by squillace** [comment]
I'll close my other; I searched all over github for this but somehow didn't find it. +23

**Comment by colrack** [comment]
I think that is the right place instead, there is more attention there, we should close this

**Comment by amarzavery** [comment]
@colrack - All the versions have been updated. Latest points to 0.9.19.

**Comment by amarzavery** [comment]
@colrack - I am fairly new to docker and havent got the time to look at this. That being said, we will target to make this an actionable item for 0.9.21 release (available around 05/04).

**Comment by colrack** [comment]
@amarzavery thanks!
up to you whether/when to close this

**Comment by amarzavery** [comment]
@colrack - For starters, I have added the Dockerfile with the latest released version to the master branch of the [azure-xplat-cli repo](https://github.com/Azure/azure-xplat-cli/blob/master/Dockerfile).
